DKTDDSwiftUI/Extensions/GreetingsExtension.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  GreetingsExtension.swift
+//  DKTDDSwiftUI
+//
+//  Created by Ramon Geronimo on 10/2/22.
+//
+
+import Foundation
+
+extension Greetings {
+    // MARK: Helpers
+    /// This helper function helps when array of names has two values and properly format the result. Ex: ["Jon", "Peter"] -> "Jon, and Peter"
+    /// It also checks if the array input has a value that has a string with two names separated with comma. Ex: ["Jon, Peter", "Cris"]
+    /// - Parameters:
+    ///   - names: String array containing two names
+    ///   - result: Names formatted string
+    func handleNamesCountEqualsTwo(_ names: inout [String], _ result: inout String) {
```

Comment:
I really think these functions would look/read better if it just returned the result

Reply:
Nice, let me make this changes too.